src/client/src/components/Navbar.jsx

```diff
@@ -1,47 +1,195 @@
+                  onChange={(e) => setQuery(e.target.value)}
+                ></input>
+                <button
+                  className="IconButton"
+                  type="submit"
+                  onClick={HandleSearch}
+                >
+                  {" "}
+                  <SearchIcon> </SearchIcon>
+                </button>
+              </Typography>
+            </div>
+            <div className="userButtons">
+              <IconButton color="inherit" onClick={redirectHome}>
+                <Box className="parentUserIconContainer">
+                  <Typography className="userIconContainer">
+                    <HomeIcon
+                      className="publicUserIcon"
+                      fontSize="large"
+                    ></HomeIcon>
+
+                    <Typography fontSize={10} variant="subtitle2">
+                      Home
+                    </Typography>
+                  </Typography>
+                </Box>
+              </IconButton>
+
+              <IconButton color="inherit">
+                <Box className="parentUserIconContainer">
+                  <Typography className="userIconContainer">
+                    <GroupIcon
+                      className="publicUserIcon"
+                      fontSize="large"
+                    ></GroupIcon>
+                    <Typography
+                      fontSize={10}
+                      className="userSubtitle"
+                      variant="subtitle2"
+                    >
+                      Network
+                    </Typography>
+                  </Typography>
+                </Box>
+              </IconButton>
+
+              <IconButton color="inherit">
+                <Box className="parentUserIconContainer">
+                  <Typography className="userIconContainer">
+                    <WorkIcon
+                      className="publicUserIcon"
+                      fontSize="large"
+                    ></WorkIcon>
+                    <Typography
+                      fontSize={10}
+                      className="userSubtitle"
+                      variant="subtitle2"
+                    >
+                      Jobs
+                    </Typography>
+                  </Typography>
+                </Box>
+              </IconButton>
+
+              <IconButton color="inherit">
+                <Box className="parentUserIconContainer">
+                  <Typography className="userIconContainer">
+                    <MessageIcon
+                      className="publicUserIcon"
+                      fontSize="large"
+                    ></MessageIcon>
+                    <Typography
+                      fontSize={10}
+                      className="userSubtitle"
+                      variant="subtitle2"
+                    >
+                      Messages
+                    </Typography>
+                  </Typography>
+                </Box>
+              </IconButton>
```

Comment:
Since these buttons don't redirect to a web page at the moment, I think we should disable them (comment them). Once we have the pages working, we can enable them again.

Reply:
commenting them out messes up the css. i think it dosen't really change anything if we comment them out or keep them in since they don't do anything, only there for aesthetic purposes. Once the pages are working, we add a onclick function that redirects to the page